src/time/format/parser.cr

```diff
@@ -13,7 +13,7 @@ struct Time::Format
       @hour = 0
       @minute = 0
       @second = 0
-      @nanosecond = 0
+      @nanosecond = 0i64
```

Comment:
This shouldn't be Int64, `Time#@nanosecond` is only Int32.

Reply:
Just looked at any code piece - `NANOSECONDS_PER_MILLISECOND = 1_000_000_i64`.

Ok, will fix :)